docs/pages/configuration/dev/files/sync.mdx

```diff
@@ -212,6 +278,8 @@ Sometimes it is useful to execute commands after the sync uploads files/director
 Make sure that post-sync commands will **<u>not</u>** trigger a new sync process which could lead to an **endless loop**.
 :::
 
+<<<<<<< HEAD
+=======
 ### `startContainer`
 
 The `startContainer` option can be used to delay starting a container until sync has finished uploading all files initially. This is very useful if your container entrypoint requires certain files. Needs `command` to be set as otherwise DevSpace doesn't know which command to start.
```

Comment:
```suggestion
The `startContainer` option can be used to delay starting a container until sync has finished uploading all files initially. This is useful if your container entrypoint requires certain files. This option requires `command` to be set; otherwise, DevSpace will not know which command to start.
```